### ISSUE TITLE
Link to codicons website?

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#162053]
Testing #161792

![image](https://user-images.githubusercontent.com/2644648/192617357-9629eada-c0e0-4f1e-8836-fd0107439ca7.png)

maybe this could link to https://microsoft.github.io/vscode-codicons/ so that users know they can use codicons?

**Comment by sandy081** [comment]
Good idea, I am planning to provide a picker to pick valid codicons instead.
